Files/Views/SettingsPages/OnStartup.xaml

```diff
@@ -62,12 +62,12 @@
                     <StackPanel 
                         Orientation="Horizontal" 
                         Margin="0,10,0,20">
-                        <TextBlock Text="Pages: " FontSize="18" />
+                        <TextBlock Text="Pages: " FontSize="18" HorizontalAlignment="Stretch" VerticalAlignment="Center" />
                         <Button 
                             x:Name="btnAddPage" 
-                            Content="Add Page" 
+                            Content="Add a new page" 
```

Comment:
Can you add localization for the new strings?

Reply:
Okay

Reply:
It has been done.